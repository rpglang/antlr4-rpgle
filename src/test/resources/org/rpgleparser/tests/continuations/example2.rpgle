     H DATFMT(
     H       *MDY&
     H       )