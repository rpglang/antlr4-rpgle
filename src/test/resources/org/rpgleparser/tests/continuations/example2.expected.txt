/*===TOKENS===*/
HS_FIXED                 <H>
ID                       <DATFMT>
'('                      <(>
ID                       <*MDY&>
')'                      <)>
/*===TREE===*/
(r
  (statement (hspec_fixed H (hs_expression DATFMT ( (hs_parm *MDY&) )) <EOF>))
)
/*======*/